### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Vedoxx**
Things accounted compared to last homework:
- camel case naming 
- blank space after keyword
- indentation

### CODE REVIEW

File: Triangle.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    protected double perimeterShape() {
+        if((this.edgeOne+this.edgeTwo+this.edgeThree)<0) throw new IllegalArgumentException("Input is not valid");
+        return this.edgeOne+this.edgeTwo+this.edgeThree;
+    }
+
+    @Override
+    protected double areaShape() {
+        double semiPerimeter=(this.edgeOne+this.edgeTwo+this.edgeTwo)/2;
+        if(semiPerimeter<0) throw new IllegalArgumentException("Input is not valid");
+        return Math.sqrt(semiPerimeter*(semiPerimeter- this.edgeOne)*(semiPerimeter- this.edgeTwo)*(semiPerimeter- this.edgeThree));
```
Comment (bjusufbe) [inline Triangle.java:22]: Interesting approach for calculating formula of triangle :)

File: seventhHomework.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class seventhHomework {
+    static Scanner object=new Scanner(System.in);
+    public static void main (String[] args) {
```
Comment (bjusufbe) [inline seventhHomework.java:7]: From what I see here, there is a possibility for user to enter N number of times each shape but calculation will always be done for each input values ? (for example: we can calculate 5 times area/perimeter for circle but every time result will be the same)

File: seventhHomework.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class seventhHomework {
+    static Scanner object=new Scanner(System.in);
```
Comment (bjusufbe) [inline seventhHomework.java:6]: Avoid using object for name of the variable

**Review by bjusufbe — COMMENTED** [review]
Good work. Generally, only change needed is to support for users to add N number of shapes in the menu, each with specific input values, before calculating area/perimeter for each of them

File: Triangle.java (hunk 1 above)
Comment (Vedoxx) [reply Triangle.java:22]: Heron's formula :)

**Pull request merged**